src/main/java/com/accantosystems/stratoss/vnfmdriver/model/alm/FindReferenceResponse.java

```diff
@@ -6,21 +6,21 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
 
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+
 
 @JsonInclude(value = JsonInclude.Include.NON_EMPTY, content = JsonInclude.Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
-@ApiModel(description = "Find reference response")
+@Schema(description = "Find reference response")
 public class FindReferenceResponse {
 
-    @ApiModelProperty(value = "Associated Topology")
+    @Schema(name = "Associated Topology")
```

Comment:
value property of @ApiModelProperty should be mapped to description in @Schema annotation. Please change all the references accordingly.